Chess/knight.h

```diff
@@ -0,0 +1,37 @@
+#ifndef H_KNIGHT
+#define H_KNIGHT
+#include "piece.h"
+#include <bits/stdc++.h>

```

Comment:
https://www.quora.com/Is-it-good-practice-to-use-include-bits-stdc++-h-in-programming-contests-instead-of-listing-a-lot-of-includes